### ISSUE TITLE
Support for Black Magic Probe ? 

### ISSUE THREAD

**Issue by kub3let** [WeActStudio/WeActStudio.MiniDebugger#1]
This looks awesome, I currently do have a weact black pill but wondering whether the mini debugger would support black magic probe as well (stm32 version) ?

**Comment by thuvasooriya** [comment]
Yeah It would be nice to have support to black magic probe or a guide on how to set it up

**Comment by kolais** [comment]
This works out of the box when following [the official guide](https://github.com/blackmagic-debug/blackmagic/tree/main/src/platforms/stlink) (assuming debugger is pre-flashed with official ST-Link firmware).
The options I chose were as per "official ST-Link V2-1". VCP also worked.
